test/lib/agent_helper.js

```diff
@@ -564,5 +565,42 @@ const helper = (module.exports = {
         this.notOk(attrs['code.column'], 'column should not exist')
       }
     })
+  },
+  /**
+   * Unwraps one or more items, revealing the original value.
+   *
+   * - `unwrap(nodule, property)`
+   * - `unwrap(func)`
+   *
+   * If called with a `nodule` and properties, the unwrapped values will be put
+   * back on the nodule. Otherwise, the unwrapped function is just returned.
+   *
+   * @param {object | Function} nodule
+   *  The source for the properties to unwrap, or a single function to unwrap.
+   * @param {string|Array.<string>} [properties]
+   *  One or more properties to unwrap. If omitted, the `nodule` parameter is
+   *  assumed to be the function to unwrap.
+   * @returns {object | Function} The first parameter after unwrapping.
+   */
+  unwrap(nodule, properties) {
```

Comment:
we still use the old unwrap all in tests so I just moved this behavior to the helper

Reply:
yes there are cases where we double wrap, mostly i think in datastore shim so we still want to unwrap all in that case which is why i put this here